FreeRTOS/FreeRTOS-Plus-TCP
# Fixed build issue for the ARP unit tests for the case when data length less than min packet

Description
-----------
Fixed build issues and updated branch to latest target branch changes

Test Steps
-----------

```
cmake -S test/unit-test -B test/unit-test/build/ 
make -C test/unit-test/build/ all -j6

```
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
